scalacheck-binding/src/main/scala/scalaz/scalacheck/ScalazProperties.scala

```diff
@@ -686,7 +695,6 @@ object ScalazProperties {
     def laws[F[_], E](implicit me: MonadError[F, E], am: Arbitrary[F[Int]], afap: Arbitrary[F[Int => Int]], aeq: Equal[F[Int]], ae: Arbitrary[E], afea: Arbitrary[E => F[Int]]): Properties =
       newProperties("monad error"){ p =>
         p.include(monad.laws[F])
```

Comment:
```diff
  p.include(monad.laws[F])
+ p.include(applicativeError.laws[F])
- p.property("raisedErrorsHandled") = raisedErrorsHandled[F, E, Int]
```

Reply:
I can't figure out where scalacheck-binding/src/main/scala/scalaz/scalacheck/ScalazProperties.scala is generated. can you please point me to it?